packages/twenty-front/src/modules/ai/components/internal/AgentChatMultipleRecordPreview.tsx

```diff
@@ -0,0 +1,96 @@
+import { ObjectMetadataItem } from '@/object-metadata/types/ObjectMetadataItem';
+import { CommandMenuContextRecordChipAvatars } from '@/command-menu/components/CommandMenuContextRecordChipAvatars';
+import { getSelectedRecordsContextText } from '@/command-menu/utils/getRecordContextText';
+import { ObjectRecord } from '@/object-record/types/ObjectRecord';
+import { MultipleAvatarChip } from 'twenty-ui/components';
+import { t } from '@lingui/core/macro';
+import { IconX, IconReload } from 'twenty-ui/display';
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { isAgentChatCurrentContextActiveState } from '@/ai/states/isAgentChatCurrentContextActiveState';
+import { useRecoilComponentStateV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentStateV2';
+
+const StyledRightIconContainer = styled.div`
+  display: flex;
+  border-left: 1px solid ${({ theme }) => theme.border.color.light};
+
+  svg {
+    cursor: pointer;
+  }
+`;
+
+const StyledChipWrapper = styled.div<{ isActive: boolean }>`
+  opacity: ${({ isActive }) => (isActive ? 1 : 0.7)};
+`;
+
+export const AgentChatMultipleRecordPreview = ({
```

Comment:
This seems quite specific to context and not what would be used for other chips?